nodes/wot-discovery.js

```diff
@@ -36,7 +38,12 @@ module.exports = function (RED) {
 
             if (coapAddresses) {
                 coapAddresses.forEach((address) => {
-                    _sendCoapDiscovery(address);
+                    if(tdURI){
+                        _sendCoapDiscovery(address, "/.well-known/wot-thing-description");
+                    }
+                    if(coreURI){
+                        _getDiscoveryLinks();
+                    }
```

Comment:
Ehem, and another very minor style fix.

```suggestion
                    if (tdURI) {
                        _sendCoapDiscovery(address, "/.well-known/wot-thing-description");
                    }
                    if (coreURI) {
                        _getDiscoveryLinks();
                    }
```